hkmc2/shared/src/test/mlscript/apps/ParseRuleVisualizerTest.mls

```diff
@@ -2,11 +2,43 @@
 
 import "../../mlscript-compile/apps/Parser.mls"
 import "../../mlscript-compile/apps/parsing/ParseRuleVisualizer.mls"
-import "../../mlscript-compile/Stack.mls"
+import "../../mlscript-compile/Iter.mls"
+import "../../mlscript-compile/XML.mls"
 
-open ParseRuleVisualizer { save, rr, render }
-open Parser { prefixRules }
-open Stack
+open ParseRuleVisualizer { render, CSS_CONTENT, filePath }
+open Parser { exprRules, moduleRules }
+open XML { html, tag, elem, style }
```

Comment:
Why not just use `open XML` in the body of `toHTML`?

Reply:
Because not only this function uses `XML`, but the following code also uses `elem` and `style`.

Reply:
(It's true that they enhance readability, which is why in the future we'll insert them automatically. But doing so manually seems a bit painful.)